syntax/supercollider.vim

```diff
@@ -39,6 +39,7 @@ syn match	scSymbol "\w\+:"
 
 syn region  scString start=+"+ skip=+\\\\\|\\"+ end=+"+
 
+syn match	scChar	"\$[^ ]"
```

Comment:
I'm not familiar with VimScript, but the standard regex to match characters is `\$\\?.`

Source: https://github.com/supercollider/supercollider/blob/master/editors/sc-ide/core/sc_lexer.cpp#L64